node/src/controllers/compute.rs

```diff
@@ -119,17 +129,29 @@ impl ComputeApi {
             Prime::Safe128Bits => EncodedModulo::U128SafePrime,
             Prime::Safe256Bits => EncodedModulo::U256SafePrime,
         };
-        Self { our_party_id, channels, prime_builder, compute_handles, ecdsa_dkg_compute_handles, services, modulo }
+        Self {
+            our_party_id,
+            channels,
+            prime_builder,
+            compute_handles,
+            ecdsa_dkg_compute_handles,
+            eddsa_dkg_compute_handles,
```

Comment:
With the above suggestion, we would assign handles only.

```suggestion
            compute_handles,
```